data/boarding missions.txt

```diff
@@ -61,6 +61,160 @@ mission "Assisting Merchant (Large)"
 
 
 
+mission "Asteroid Course A"
+	assisting
+	minor
+	to offer
+		random < 10
+	source
+		government "Merchant" "Syndicate"
+		category "Light Freighter" "Interceptor"
+		not attributes "automaton"
+	destination "Earth"
+	on offer
+		conversation
+			`As you are approaching the <origin>, your radar sensors detect that the ship you are assisting is in the middle of a large asteroid's course! While in a normal flight this wouldn't cause much trouble, a ship without active shields and that has already suffered heavy damages could be completely destroyed.`
+			branch unarmed
+				has "armament deterrence" == 0
```

Comment:
This condition should get flagged as unrecognized / invalid. Either `not "armament deterrence"` or `armament deterrence == 0` is what you want.

Reply:
Weird, the 'has "armament deterrence" == 0' worked fine on my end and gave no error. I know it applies to all the fleet, but that was the best I could think of and it works decently for what I need here.
Removed the "has" anyways